core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -6546,6 +6546,16 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
             memset(frame_lp + cur_func->param_cell_num, 0,
                    (uint32)(cur_func->local_cell_num * 4));
 
+#if WASM_ENABLE_REF_TYPES != 0 && WASM_ENABLE_GC == 0
+            /* externref/funcref should be NULL_REF rather than 0 */
+            for (i = 0; i < cur_func->local_cell_num; i++) {
+                if (cur_wasm_func->local_types[i] == VALUE_TYPE_EXTERNREF
+                    || cur_wasm_func->local_types[i] == VALUE_TYPE_FUNCREF) {
+                    *(frame_lp + cur_func->param_cell_num + i) = NULL_REF;
+                }
+            }
```

Comment:
a local may occupies several cells, so cur_func->local_cell_num may be larger than cur_wasm_func->local_count.
Had better be like:
```C
    local_cell_idx = cur_func->param_cell_num;
    for (i = 0; i < cur_wasm_func->local_count; i++) {
        if (cur_wasm_func->local_types[i] == VALUE_TYPE_EXTERNREF
            || cur_wasm_func->local_types[i] == VALUE_TYPE_FUNCREF) {
            *(frame_lp + local_cell_idx) = NULL_REF;
        }
        local_cell_idx += wasm_value_type_cell_num(cur_wasm_func->local_types[i]);
    }
```